helpers/constants.js

```diff
@@ -118,8 +99,7 @@ var constants = {
 		// Testnet
 		'da3ed6a45429278bac2666961289ca17ad86595d33b31037615d4b8e8f158bba',
 	],
-	numberLength: 100000000,
-	requestLength: 104,
+	normalizer: 100000000,
```

Comment:
`normalizer` means nothing, please use better name.

Reply:
@diego-G @4miners any suggestions for the name?

Reply:
@ManuGowda be creative 😃

Reply:
`normalizer` is ok to me. It was like that before